templates/AureliaSpa/webpack.config.js

```diff
@@ -1,45 +1,52 @@
-var isDevBuild = process.argv.indexOf('--env.prod') < 0;
-var path = require('path');
-var webpack = require('webpack');
-var AureliaWebpackPlugin = require('aurelia-webpack-plugin');
+const path = require('path');
+const webpack = require('webpack');
+const {AureliaPlugin} = require('aurelia-webpack-plugin');
 
-var bundleOutputDir = './wwwroot/dist';
-module.exports = {
-    resolve: { extensions: [ '.js', '.ts' ] },
-    entry: { 'app': 'aurelia-bootstrapper-webpack' }, // Note: The aurelia-webpack-plugin will add your app's modules to this bundle automatically
-    output: {
-        path: path.resolve(bundleOutputDir),
-        publicPath: '/dist',
-        filename: '[name].js'
-    },
-    module: {
-        loaders: [
-            { test: /\.ts$/, include: /ClientApp/, loader: 'ts-loader', query: { silent: true } },
-            { test: /\.html$/, loader: 'html-loader' },
-            { test: /\.css$/, loaders: [ 'style-loader', 'css-loader' ] },
-            { test: /\.(png|woff|woff2|eot|ttf|svg)$/, loader: 'url-loader?limit=100000' },
-            { test: /\.json$/, loader: 'json-loader' }
+module.exports = (env) => {
```

Comment:
You can set a default empty object `(env = {}) =>` here and you won't have to `env && ` every time it's used :) Also, it might be arguably better to use destructuring here (maybe even with reassignment), like: `({prod} = {})`.